vendor_accounts.yaml

```diff
@@ -187,4 +187,10 @@
   accounts: ['218546966473']
 - name: 'Emnify'
   source: 'https://www.emnify.com/datastreamer-integration-into-aws'
-  accounts: ['884047677700']
\ No newline at end of file
+  accounts: ['884047677700']
+- name: 'Cloudaware'
+  source: 'https://docs.cloudaware.com/DOCS/AWS-Start-Guide.1171685377.html'
```

Comment:
This document doesn't indicate this account number. I'm hesitant about adding accounts from vendors that they are not publicly associating themselves with because:
1) Maybe that vendor wants to keep their account number private (not a great reason)
2) I'm worried we'll add an incorrect account ID and so perhaps an attacker one day tells me to add an account that is their own and not some vendors, and then a security team ignores it if they get breached somehow one day from that attacker's account.